core/templates/dev/head/pages/profile/profile.html

```diff
@@ -29,7 +29,7 @@ <h1 class="oppia-profile-h1">{{PROFILE_USERNAME}}</h1>
     <md-card class="oppia-profile-user-card">
       <div class="oppia-profile-picture-container">
         <div ng-if="profileIsOfCurrentUser" class="oppia-profile-picture">
-          <a href="/preferences" class="oppia-profile-prefererences">
+          <a href="/preferences" class="Oppia-profile-preferences">
```

Comment:
This is wrong, why capitalize?

(though thanks for fixing the spelling!)

Reply:
Done, thanks!